### DISCUSSION TITLE
Ignore Ghost ? What is this?

### DISCUSSION THREAD

**Discussion by gitttt54** [meisnate12/Plex-Meta-Manager#1045] [Q&A] [answered]
Hello,

would someone please elaborate [Ignore Ghost](https://metamanager.wiki/en/latest/home/environmental.html?highlight=ghosts#ignore-ghost) from the wiki to me?

I can't see any difference on the console (or log) starting pmm like :

`docker exec -it pmm python plex_meta_manager.py --run -c config/config_mm.yml --no-missing --ignore-ghost`

or 

`docker exec -it pmm python plex_meta_manager.py --run -c config/config_mm.yml --no-missing`

Thank you!

**Comment by meisnate12** [comment, marked as answer]
theres some instances where pmm outputs what its currently doing and updates it on the same line without saving that line to the meta.log `--ignore-ghost` causes those not to print

**Reply by gitttt54** [reply]
I see, thank you. 
Now I recognized, that when those updates on the same line are missing, it looks on large operations that nothing is happening 8-)
